package/MDAnalysis/coordinates/RDKit.py

```diff
@@ -95,10 +134,163 @@ def __init__(self, filename, **kwargs):
+
+        for i, (atom, element) in enumerate(zip(ag, elements)):
+            # create atom
+            rdatom = Chem.Atom(element)
+            # disable adding H to the molecule
+            rdatom.SetNoImplicit(True)
+            # add PDB-like properties
+            mi = Chem.AtomPDBResidueInfo()
+            for attr, values in pdb_attrs.items():
+                _add_mda_attr_to_rdkit(attr, values[i], mi)
+            rdatom.SetMonomerInfo(mi)
+            # other properties
+            for attr in other_attrs.keys():
+                value = other_attrs[attr][i]
+                attr = attr[:-1] # plural to singular
+                if isinstance(value, np.float):
+                    rdatom.SetDoubleProp("_MDAnalysis_%s" % attr, float(value))
+                elif isinstance(value, np.int):
+                    rdatom.SetIntProp("_MDAnalysis_%s" % attr, int(value))
+                else:
+                    rdatom.SetProp("_MDAnalysis_%s" % attr, value)
+            # add atom
+            index = mol.AddAtom(rdatom)
+            # map index in universe to index in mol
+            atom_mapper[atom.ix] = index
+
+        try:
+            bonds = ag.bonds
+            if (len(bonds) == 0) and (ag.n_atoms > 1):
+                # force guessing bonds
+                raise NoDataError
+        except NoDataError:
+            warnings.warn(
+                "No `bonds` attribute in this AtomGroup. Guessing bonds based"
+                "on atoms coordinates")
+            ag.guess_bonds()
+            bonds = ag.bonds
+
+        for bond in bonds:
+            try:
+                bond_indices = [atom_mapper[i] for i in bond.indices]
+            except KeyError:
+                # one of the atoms of the bond is not part of the atomgroup
+                continue
+            try:
+                bond_type = bond.type.upper()
+            except AttributeError:
+                # bond type can be a tuple for PDB files
+                bond_type = None
+            bond_type = RDBONDTYPE.get(bond_type, RDBONDORDER.get(
+                bond.order, Chem.BondType.SINGLE))
+            mol.AddBond(*bond_indices, bond_type)
+
+        # sanitization
+        Chem.SanitizeMol(mol)
+
+        return mol
+
+
+def _add_mda_attr_to_rdkit(attr, value, mi):
+    """Converts an MDAnalysis atom attribute into the RDKit equivalent and 
+    stores it into an RDKit AtomPDBResidueInfo object.
+
+    Parameters
+    ----------
+
+    attr : str
+        Name of the atom attribute in MDAnalysis in the singular form
+    value : object, np.int or np.float
+        Attribute value as found in the AtomGroup
+    mi : rdkit.Chem.rdchem.AtomPDBResidueInfo
+        MonomerInfo object that will store the relevant atom attributes
+    """
+    if isinstance(value, np.generic):
+        # convert numpy types to python standard types
+        value = value.item()
+    if attr == "names":
+        # RDKit needs the name to be properly formated for a
```

Comment:
@jbarnoud didn't you already write code to this effect in the PDBWriter? Or was that for parsing still?+

Reply:
Also, less useful comment than Richard's, but `formated`->`formatted` I think